crates/consensus/src/engine/mod.rs

```diff
@@ -72,16 +78,16 @@ pub enum EngineMessage {
 /// The consensus engine API implementation
 #[must_use = "EthConsensusEngine does nothing unless polled."]
 pub struct EthConsensusEngine<Client> {
+    client: Arc<Client>,
     /// Consensus configuration
     config: Config,
-    client: Arc<Client>,
-    /// Placeholder for storing future blocks
-    local_store: HashMap<H64, ExecutionPayload>, // TODO: bound
-    // remote_store: HashMap<H64, ExecutionPayload>,
     rx: UnboundedReceiverStream<EngineMessage>,
+    // TODO: Placeholder for storing future blocks. Make cache bounded.
```

Comment:
Lru crate

Reply:
we can switch once we start actually using it. left a comment